.github/workflows/macos-qt.yml

```
@@ -49,13 +60,13 @@ jobs:
     - name: Package
       run: |
         mkdir upload
-        mv ${{github.workspace}}/build/shadps4.app upload
-        mv ${{github.workspace}}/build/translations upload
+        cp -R ${{github.workspace}}/build/shadps4.app upload
+        cp -R ${{github.workspace}}/build/translations upload
```

Copying files at the end resulted in bigger cache files and a worst build time, I am considering reverting `cp -R` to `mv` in macOS/-qt and Windows/-qt.
```suggestion
        mv ${{github.workspace}}/build/shadps4.app upload
        mv ${{github.workspace}}/build/translations upload
```